lib/scala/pkg/src/test/resources/Enso_Project_Archive_Test/package.yaml

```diff
@@ -4,3 +4,4 @@ enso-version: default
 version: "0.0.1"
 author: "Enso Team <contact@enso.org>"
 maintainer: "Enso Team <contact@enso.org>"
+jvm: false
```

Comment:
Why do we need `jvm: false` here? Isn't `false` the default when there is no `jvm:`?